examples/redis-operator.md

```diff
@@ -0,0 +1,109 @@
+    pmem-csi-node-dwws2           2/2     Running   0          125m
+    pmem-csi-node-kj7gs           2/2     Running   0          125m
+    ```
+
+## Redis operator installation and Redis cluster deployment
+The steps to install the Redis operator by Spotahome are listed below:
+ 1. Clone the [redis-operator](https://github.com/spotahome/redis-operator) project into `$GOPATH/src/github.com/spotahome`
+ 2. Install the Redis operator and validate that the `CRD` was properly defined:
+    ```sh
+    $ cd $GOPATH/src/github.com/spotahome/redis-operator
+    $ kubectl create -f example/operator/all-redis-operator-resources.yaml
+
+    $ kubectl get po
+    NAME                             READY   STATUS    RESTARTS   AGE
+    pmem-csi-controller-0            3/3     Running   0          134m
+    pmem-csi-node-c82p5              2/2     Running   0          133m
+    pmem-csi-node-dwws2              2/2     Running   0          134m
+    pmem-csi-node-kj7gs              2/2     Running   0          134m
+    redisoperator-688f6f4fcc-lv5h5   1/1     Running   0          7m29s
+
+    $ kubectl get crd
+    NAME                                     CREATED AT
+    csidrivers.csi.storage.k8s.io            2019-06-21T16:55:46Z
+    csinodeinfos.csi.storage.k8s.io          2019-06-21T16:55:47Z
+    redisfailovers.databases.spotahome.com   2019-06-21T19:03:36Z
+    ```
+3. Deploy a Redis cluster that uses PMEM-CSI:
+    ```sh
+    $ kubectl create -f example/redisfailover/pmem.yaml
+    $ kubectl get po
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    pmem-csi-controller-0                     3/3     Running   0          145m
+    pmem-csi-node-c82p5                       2/2     Running   0          144m
+    pmem-csi-node-dwws2                       2/2     Running   0          145m
+    pmem-csi-node-kj7gs                       2/2     Running   0          145m
+    redisoperator-688f6f4fcc-lv5h5            1/1     Running   0          18m
+    rfr-redisfailover-pmem-0                  1/1     Running   0          9m37s
+    rfr-redisfailover-pmem-1                  1/1     Running   0          6m28s
+    rfr-redisfailover-pmem-2                  1/1     Running   0          3m15s
+    rfs-redisfailover-pmem-7595857d4c-56mmz   1/1     Running   0          9m37s
+    rfs-redisfailover-pmem-7595857d4c-c69xg   1/1     Running   0          9m37s
+    rfs-redisfailover-pmem-7595857d4c-cqx4p   1/1     Running   0          9m37s
+    ```
+    **Note 1**: How to verify that the Redis instances are using the volumes provided by `pmem-csi-sc-ext4`? You can match the volumes claim name used by each Redis instance, and then match it to the current `pvc` which gets provisioned  by `pmem-csi-sc-ext4`, those needs to be properly bound. Finally, you can check the block devices in the worker nodes of your K8s cluster and match the ones under `pmem` block device, i.e.:
```

Comment:
Replace with this text:
### Verify that the Redis instances are using the provided volumes 
First, match the volumes claim name used by each Redis instance to the current `pvc` which is provisioned by `pmem-csi-sc-ext4`. Both items must be properly bound. 
Next, check the block devices in the worker nodes of your K8S cluster and match the ones under the `pmem` block device, as shown below.